.pre-commit-config.yaml

```diff
@@ -1,14 +1,14 @@
 repos:
-  - repo: git://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.0.1
     hooks:
       - id: check-merge-conflict
       - id: check-yaml
       - id: detect-private-key
       - id: trailing-whitespace
 
-  - repo: git://github.com/igorshubovych/markdownlint-cli
-    rev: v0.28.1
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.29.0
```

Comment:
I ran into problems trying to run pre-commit with the `git://` URL:

```
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
```

The error message pointed to [this article](https://github.blog/2021-09-01-improving-git-protocol-security-github/).  It seemed we needed to migrate over to `https://` instead, so I did that.  I also updated to the latest version of these linters.

Reply:
The main "git://" vs "https://" issue was fixed upstream in the main branch (just merged it in), but I did include a couple of minor version updates since then.